staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -41,7 +41,7 @@ const (
 
 // FailurePolicyType specifies a failure policy that defines how unrecognized errors from the admission endpoint are handled.
 // +enum
-type FailurePolicyType string
+type FailurePolicyType = v1.FailurePolicyType
 
 const (
```

Comment:
Since we type alias to v1.FailurePolicyType, v1.Ignore and v1.Fail is needed in below as well.

Reply:
So you're suggesting I do this PR and merge it and then in a separate PR do `./hack/update-codegen.sh`?

Reply:
done